tools/js-tools/load-eslint-ignore.js

```diff
@@ -98,8 +103,9 @@ function loadIgnorePatterns( basedir ) {
 						}
 					}
 					if ( b ) {
-						debugrp( ` Accepted ${ l } as ${ n + '/' + b }` );
-						rules.push( n + '/' + b );
+						const rule = b.replace( /^\/+/, '' );
+						debugrp( ` Accepted ${ l } as ${ n + rule }` );
+						rules.push( n + rule );
```

Comment:
The earlier instance seems correct, but are we missing a `/` in this one?

```suggestion
						debugrp( ` Accepted ${ l } as ${ n + '/' rule }` );
						rules.push( n + '/' + rule );
```

Reply:
Nope, that was intentionally removed. They switched from [ignore](https://www.npmjs.com/package/ignore) to [minimatch](https://www.npmjs.com/package/minimatch) and that's one of the behavioral differences: in `.gitignore` files an entry without a slash is implicitly preceded with `**/` and a leading slash can be used to explicitly anchor it, while with minimatch it's always anchored and you need to explicitly lead with `**/` if you want unanchored behavior, and it doesn't ignore a leading `/` in the pattern.

There are probably some other edge cases due to that switch, but I didn't dig too deeply beyond checking that it's linting all the same files as it was before.

P.S. Another difference, which is not due to the library change, is that eslint no longer default-ignores dotfiles, so `projects/packages/search/.size-limit.js` is now linted.